### ISSUE TITLE
Process 'ForkPoolWorker-2' error

### ISSUE THREAD

**Issue by ericman93** [pandas-profiling/pandas-profiling#237]
**Describe the bug**

Running profiling from celery I have the following error
```
[2019-08-14 01:44:31,087: ERROR/MainProcess] Process 'ForkPoolWorker-2' pid:6237 exited with 'signal 11 (SIGSEGV)'
2019-08-14 01:44:31,114 - 6228 - celery.worker.request - ERROR - Task handler raised error: WorkerLostError('Worker exited prematurely: signal 11 (SIGSEGV).',)
Traceback (most recent call last):
  File "/Users/eric/.local/share/virtualenvs/romee-vHL_yubm/lib/python3.6/site-packages/billiard/pool.py", line 1226, in mark_as_worker_lost
    human_status(exitcode)),
billiard.exceptions.WorkerLostError: Worker exited prematurely: signal 11 (SIGSEGV).
```
After adding logs I know that the error is in `get_missing_diagrams` method somewhere after creating `missing_map`
But when returning empty dict instead of running the method the error still reproduce, but somewhere else

I moved to celery after I couldn't run profiling on my gunicron hosted flask thanks to the next error
```
Exception in thread Thread-6:
Traceback (most recent call last):
  File "/Users/eric/.pyenv/versions/3.6.2/lib/python3.6/threading.py", line 916, in _bootstrap_inner
    self.run()
  File "/Users/eric/.pyenv/versions/3.6.2/lib/python3.6/threading.py", line 864, in run
    self._target(*self._args, **self._kwargs)
  File "/Users/eric/.pyenv/versions/3.6.2/lib/python3.6/multiprocessing/pool.py", line 420, in _handle_tasks
    if thread._state:
AttributeError: '_MainThread' object has no attribute '_state'
```
I guess it's related to the fact that I'm using eventlet's monkey matching 

**To Reproduce**
Run profiling from celery

_Data:_
```
{"Rooms":{"count":15064.0,"mean":2.932488051,"std":0.9629786668,"min":1.0,"25%":2.0,"50%":3.0,"75%":3.0,"max":12.0},"Price":{"count":15064.0,"mean":1067009.8032395113,"std":639212.2773815426,"min":85000.0,"25%":640000.0,"50%":900000.0,"75%":1320000.0,"max":9000000.0},"Bedroom2":{"count":15064.0,"mean":2.8821030271,"std":1.0044254125,"min":0.0,"25%":2.0,"50%":3.0,"75%":3.0,"max":20.0},"Car":{"count":15064.0,"mean":1.5866967605,"std":0.9713677657,"min":0.0,"25%":1.0,"50%":2.0,"75%":2.0,"max":10.0},"Landsize":{"count":15064.0,"mean":503.4297663303,"std":3792.6445568356,"min":0.0,"25%":102.0,"50%":360.0,"75%":634.0,"max":433014.0},"YearBuilt":{"count":15064.0,"mean":1167.977496017,"std":965.8051217776,"min":0.0,"25%":0.0,"50%":1925.0,"75%":1975.0,"max":2018.0},"Lattitude":{"count":15064.0,"mean":-37.8098487854,"std":0.081152387,"min":-38.1825485229,"25%":-37.8581008911,"50%":-37.8036251068,"75%":-37.7562704086,"max":-37.408531189},"Longtitude":{"count":15064.0,"mean":144.9963531494,"std":0.1063751578,"min":144.4318084717,"25%":144.9311904907,"50%":145.0009231567,"75%":145.0599975586,"max":145.5263519287}}
```
_Code:_ Preferably, use this code format:
```python
"""
Test for issue XXX:
https://github.com/pandas-profiling/pandas-profiling/issues/XXX
"""
import pandas as pd
import pandas_profiling


def test_issueXXX():
    df = pd.read_csv(r'<file>')
    profile = df.profile_report(title=f'Melbourne Profiling Report', style={'primary_color': "#23a696"})
```

**Version information:**

Version information is essential in reproducing and resolving bugs. Please report:

* _Python version_: 3.6.2.
* _Environment_: Celery
<details><summary>Click to expand <strong><em>Version information</em></strong></summary>
<p>

alembic==0.9.5
amqp==2.5.0
aniso8601==7.0.0
appnope==0.1.0 ; sys_platform == 'darwin'
asn1crypto==0.24.0
astroid==2.2.5
astropy==3.2.1
atomicwrites==1.3.0
attrs==18.1.0
authlib==0.11
backcall==0.1.0
bcrypt==3.1.3
beautifulsoup4==4.4.1
billiard==3.6.0.0
bleach==3.1.0
blinker==1.4
boto3==1.9.169
botocore==1.12.207
cached-property==1.5.1
cachetools==3.1.1
celery==4.3.0
certifi==2019.6.16
cffi==1.11.0
chardet==3.0.4
click-plugins==1.1.1
click==7.0
cligj==0.5.0
cloudpickle==1.2.1
configparser==3.5.0
confuse==1.0.0
coverage==4.4.1
cryptography==2.5
cycler==0.10.0
cython==0.26.1
dask[complete]==2.2.0
decorator==4.1.2
defusedxml==0.6.0
descartes==1.1.0
distributed==2.2.0
distro==1.0.4
docker-compose==1.24.1
docker-pycreds==0.4.0
docker[ssh]==3.7.3
dockerpty==0.4.1
docopt==0.6.2
docutils==0.14
dominate==2.4.0
ecdsa==0.13.2
eli5==0.8
entrypoints==0.3
eventlet==0.22.1
faker==0.9.0
fastparquet==0.2.1
fiona==1.8.6
flask-bcrypt==0.7.1
flask-bootstrap==3.3.7.1
flask-cors==3.0.3
flask-debugtoolbar==0.10.1
flask-login==0.4.1
flask-redis==0.3.0
flask-restful==0.3.6
flask-script==2.0.6
flask-socketio==3.1.2
flask-sqlalchemy==2.3.2
flask-testing==0.6.2
flask==0.12.2
fsspec==0.4.1
funcsigs==1.0.2
future==0.17.1
futures==3.1.1
geocoder==1.38.1
geopandas==0.3.0
gevent==1.4.0
google-api-core==1.14.2
google-auth-oauthlib==0.4.0
google-auth==1.6.3
google-cloud-bigquery==1.12.0
google-cloud-core==1.0.3
google-cloud-storage==1.16.0
google-resumable-media==0.3.2
googleapis-common-protos==1.6.0
goslate==1.5.1
graphviz==0.11.1
greenlet==0.4.15 ; platform_python_implementation == 'CPython'
gunicorn==19.9.0
h3==3.4.3
heapdict==1.0.0
html5lib==0.9999999
htmlmin==0.1.12
httplib2==0.9.1
idna==2.7
importlib-metadata==0.19
ipykernel==5.1.2
ipython-genutils==0.2.0
ipython==7.7.0
isort==4.3.21
itsdangerous==1.1.0
jedi==0.15.1
jinja2==2.9.6
jmespath==0.9.4
joblib==0.11
jsonschema==2.6.0
jupyter-client==5.3.1
jupyter-core==4.5.0
kiwisolver==1.1.0
kombu==4.6.3
lazy-object-proxy==1.4.1
llvmlite==0.29.0
locket==0.2.0
logzio-python-handler==2.0.10
lxml==4.1.1
mako==1.1.0
markdown==2.6.9
markupsafe==1.1.1
matplotlib==3.1.1
mccabe==0.6.1
minio==3.0.4
missingno==0.4.2
mistune==0.8.4
more-itertools==7.2.0
msgpack==0.6.1
munch==2.3.2
nbconvert==5.6.0
nbformat==4.4.0
notebook==6.0.0
numba==0.45.1
numexpr==2.6.9
numpy==1.15.4
oauth==1.0.1
oauthlib==3.0.0
packaging==19.1
pandas-gbq==0.6.1
pandas-profiling==2.3.0
pandas==0.23.4
pandasql==0.7.3
pandocfilters==1.4.2
paramiko==2.6.0
parso==0.5.1
partd==1.0.0
passlib==1.7.1
patsy==0.5.1
pexpect==4.7.0 ; sys_platform != 'win32'
phik==0.9.8
pickleshare==0.7.5
pillow==6.1.0
pluggy==0.12.0
prometheus-client==0.7.1
prompt-toolkit==2.0.9
protobuf3-to-dict==0.1.5
protobuf==3.9.1
psutil==5.6.3
psycopg2==2.7.3.2
ptyprocess==0.6.0 ; os_name != 'nt'
py-healthcheck==1.9.0
py==1.8.0
pyasn1-modules==0.2.6
pyasn1==0.4.6
pycparser==2.19
pycryptodome==3.8.2
pydictionary==1.5.2
pygments==2.4.2
pyhive==0.6.1
pylint==2.3.1
pymysql==0.9.2
pynacl==1.3.0
pyodbc==4.0.22
pyparsing==2.4.2
pyproj==2.2.1
pytest-pylint==0.14.1
pytest-runner==5.1
pytest==5.0.1
python-dateutil==2.6.1
python-debian==0.1.31
python-editor==1.0.4
python-engineio==3.8.2.post1
python-jose==2.0.2
python-socketio==4.2.0
pytz==2017.2
pyyaml==3.13
pyzmq==18.0.1
ratelim==0.1.6
redis==2.10.6
requests-futures==0.9.9
requests-oauthlib==1.2.0
requests==2.20.0
rsa==4.0
rtree==0.8.3
s3fs==0.3.3
s3transfer==0.2.1
sagemaker==1.36.4
scikit-learn==0.20.1
scipy==1.3.0
seaborn==0.9.0
send2trash==1.5.0
sentry-sdk==0.7.3
shapely==1.6.4.post2
singledispatch==3.4.0.3
six==1.12.0
sklearn==0.0
sortedcontainers==2.0.3
sqlalchemy-redshift==0.7.3
sqlalchemy==1.2.12
statsmodels==0.10.1
tables==3.4.2
tabulate==0.8.3
tblib==1.4.0
teradatasql==16.20.0.45
teradatasqlalchemy==16.20.0.4
terminado==0.8.2
testpath==0.4.2
text-unidecode==1.2
texttable==0.9.1
thrift==0.11.0
toolz==0.10.0
tornado==5.1.1
tqdm==4.33.0
traitlets==4.3.2
tsfresh==0.12.0
typed-ast==1.4.0 ; implementation_name == 'cpython'
typing==3.7.4
urllib3==1.22
vine==1.3.0
visitor==0.1.3
wcwidth==0.1.7
webencodings==0.5.1
websocket-client==0.56.0
werkzeug==0.12.2
wrapt==1.11.2
xgboost==0.90
xlrd==1.1.0
xmljson==0.1.9
zict==1.0.0
zipp==0.5.2
zope.interface==4.1.3
</p>
</details>

**Comment by ericman93** [comment]
when running this on ubuntu server everything worked fine
only reproduce on my mac (10.14.6)

**Comment by sbrugman** [comment]
We should include mac in the travis CI then...
